lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -41,7 +41,6 @@ bleach = "*"
 "bz2file" = {version = "*", markers = "python_version < '3.3'"}
 ipaddress = {version = "*", markers = "python_version < '3.3'"}
 boltons = "*"
-gunicorn = "*"
```

Comment:
@mvdbeek This should actually be moved in the `[dev-packages]` section above, not deleted.

Reply:
And I was surprised that I didn't find a Pipfile in develop ...

Reply:
The `develop` subdir is gone in 18.09, commit a2d99e30c727e7050c1fd6218603b7b2941c8c4d .

Reply:
@mvdbeek Same thing confused me briefly when I swapped nodeenv around to a core requirement!